ros_ws/src/get_user_request.py

```diff
@@ -0,0 +1,141 @@
+import os
+import langchain
+
+from langchain.llms import OpenAI
+from langchain import PromptTemplate
+from langchain import schema
+from langchain.prompts import (ChatPromptTemplate,PromptTemplate,SystemMessagePromptTemplate, HumanMessagePromptTemplate)
+from langchain.chat_models import ChatOpenAI
+from langchain.chains import LLMChain
+from langchain.schema import HumanMessage, SystemMessage, AIMessage
+api_key=os.environ["OPENAI_API_KEY"]
```

Comment:
Constants should be all caps, a-la `API_KEY`. Also, probably follow the pattern I demonstrated via the `providers.py` in the other LLM service, where an API key can be passed in or loaded from default via the env var. It allows easier config and debugging later.

Reply:
Fixed API_KEY